likeminds-chat-reactnative-integration/ChatSX/context/ChatroomContext.tsx

```diff
@@ -695,7 +695,7 @@ export const ChatroomContextProvider = ({ children }: ChatroomContextProps) => {
       if (navigationFromNotification) {
         if (appState.match(/active|foreground/)) {
           // App has gone to the background
-          await fetchInitAPI();
+          // await fetchInitAPI();
```

Comment:
remove unnecessary code, if it is not used in future

Reply:
left it for future reference in case of any bug or weird behaviour